public/myScript/orders.js

```diff
@@ -0,0 +1,158 @@
+
+
+var dynamicTable = (function() {
+
+    var _tableId, _table,
+        _fields, _headers,
+        _defaultText;
+
+    /** Builds the row with columns from the specified names.
+     *  If the item parameter is specified, the memebers of the names array will be used as property names of the item; otherwise they will be directly parsed as text.
+     */
+
+    function format(item){
+            var tagcolor = '<tr>';
+            switch (item){
+                case 4:tagcolor = '<tr bgcolor = " #00FF7F">';break;
+                case 2:
+                case 3:tagcolor = '<tr bgcolor = "#FFFF00">';break;
+                case 0:
+                case 1:tagcolor = '<tr bgcolor = "#FF0000">';break;
+                default: tagcolor = '<tr>'; break
+            }
+            return tagcolor;
+        }
+
+    function translate(item){
+        var tagcolor = '<tr>';
```

Comment:
Avoid mutating function params, you are passing a number here and expecting a string in result, whihc is fine. But in order to achieve that you should return a new variable, not mutate existing.